docker-app/qfieldcloud/urls.py

```diff
@@ -79,5 +91,14 @@
     path("auth/", include("rest_framework.urls")),
     path("accounts/", include("allauth.urls")),
     re_path(r"^invitations/", include("invitations.urls", namespace="invitations")),
+
+   
+    
+    # # ADDED
+    # path("organizations/<str:organization_name>/team/', TeamListCreateView.as_view(), name='team-list-create'),
+    
     path("__debug__/", include("debug_toolbar.urls")),
+   
+   
+
```

Comment:
```suggestion
    re_path(r"^invitations/", include("invitations.urls", namespace="invitations")),
    path("__debug__/", include("debug_toolbar.urls")),
```

We can remove all this noise.

Reply:
Removed!